src/lib/containers/download_list_v2/CreatePackageV2.tsx

```diff
@@ -131,16 +111,14 @@ export const CreatePackageV2 = (props: CreatePackageV2Props) => {
           )}
         </div>
       </div>
-      {alert.message && (
-        <Alert
-          transition={false}
-          variant={alert.variant}
-          show={true}
-          className={alert.className}
-        >
-          {alert.message}
-        </Alert>
-      )}
+      <FullWidthAlert
```

Comment:
Change from inline Alert to a global alert.

Reply:
This works. Any particular reason why using the component instead of calling `displayToast`?